component-css/side-nav.css

```diff
@@ -0,0 +1,233 @@
+*{
+    box-sizing: border-box;
+    margin: 0px;
+    padding: 0px;
+}
+
+:root{
+    --background-color:#F8F8FF;
+    --banner-text-color: #dc2626;
+    --main-primary-color: #00008b;
+    --main-font : 'Raleway', sans-serif;
+    --primary:#1e8be4;
+    --secondary:#778899;
+    --success:#129212;
+    --danger: #ca160d;
+    --warning:#FFCC00;
+    --info : indigo;
+    --light: whitesmoke;
+    --dark: #000000;
+  
+    --primary-hover:#106fbd;
+    --secondary-hover:#697683;
+    --success-hover:#017201;
+    --danger-hover: #b32922;
+    --warning-hover:#d6ab00;
+    --info-hover : rgb(55, 2, 94);
+    --light-hover: rgb(212, 201, 201);
+    --dark-hover: #000000;
+}
+
+
+
+
+
+/*                                 SIDEBAR                                      */
+.heading{
+    width: 100%;
+    padding: 0.9rem;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+}
+.heading .brand-name img{
+    width: 160px;
+}
+
+.sideBar{
```

Comment:
don't write css classnames in camelcase, generally it's written in lowercase. I guess it's a typo mistake.

Reply:
I have done necessary changes as you suggested and thanks a lot for reviewing my code